docs/en/notes/projects.md

```diff
@@ -0,0 +1,10 @@
+# Projects
+
+We list some open-source projects built on MMClassification. They are good examples of how to use MMClassification as a library, to help your own projects.
+
+- [OpenMixup](https://github.com/Westlake-AI/openmixup): an open-source toolbox for supervised, self-, and semi-supervised visual representation learning with mixup based on PyTorch, especially for mixup-related methods.
+- [Involution](https://github.com/d-li14/involution): Inverting the Inherence of Convolution for Visual Recognition, a brand new neural operator.
+- [AI Power](https://github.com/ykk648/AI_power): AI toolbox and pretrain models.
+- [OpenBioSeq](https://github.com/Westlake-AI/OpenBioSeq): an open-source supervised and self-supervised bio-sequence representation learning toolbox based on PyTorch.
```

Comment:
- [ConvolutionOfConvolution(CVPR2022)](https://github.com/Genera1Z/ConvolutionOfConvolution):  an algorithm that associates kernels in a layer and lets them collaborate spatially.

this work can be added here too.